index.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+session_start();
+include_once 'config.php';
+include_once 'functions.php';
+
+$query = $db->prepare('SELECT `name`, `latin_name`, `hardiness_type`, `image` FROM `details` 
+    JOIN `hardiness_types` ON `details`.`hardiness`=`hardiness_types`.`id`');
+$query->execute();
+$plants = $query->fetchAll();
+
+//Checking if the query worked:
```

Comment:
Again, commented out code.
Useful while your writing and debugging your code, but should be removed before you put stuff into GitHub.